agent/agent_method_script.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
```

Comment:
Let's add `set -o xtrace` here, at least for now?

Reply:
Done